shell/platform/windows/text_input_plugin.h

```diff
@@ -85,7 +104,7 @@ class TextInputPlugin {
   // as TextEditingDeltas or as one TextEditingValue.
   // For more information on the delta model, see:
   // https://master-api.flutter.dev/flutter/services/TextInputConfiguration/enableDeltaModel.html
-  bool enable_delta_model;
+  bool enable_delta_model = false;
```

Comment:
The lack of default value caused optimized unit tests to send delta model messages unexpectedly.

Reply:
Strange - this should be default zero-initialized. Do we know in which specific situations there were problems?

fwiw even if only for readability as a declaration of intent, we should make this change.